ui/apps/platform/src/utils/pdfUtils.js

```diff
@@ -128,7 +128,7 @@ const createPDFTable = (tableData, entityType, query, pdfId, tableColumns) => {
                             String(flattenedObj[key]).replace(/\s+/g, ' ').trim()) ||
                         'N/A';
                 }
-                td.innerHTML = colValue;
+                td.innerHTML = colValue.replace(/<\/?[^>]+(>|$)/g, "");
```

Comment:
To appease the linter
```suggestion
                td.innerHTML = colValue.replace(/<\/?[^>]+(>|$)/g, '');
```